libsoftwarecontainer/src/container.cpp

```diff
@@ -766,6 +796,11 @@ std::string Container::gatewaysDirInContainer() const
     return GATEWAYS_PATH;
 }
 
+std::string Container::containerRoot() const
+{
+    return m_containerRoot + "/" + id();
+}
+
 std::string Container::gatewaysDir() const
 {
     return m_containerRoot + "/" + id() + GATEWAYS_PATH;
```

Comment:
maybe it could be good idea to use containerRoot() here instead of `m_containerRoot + "/" + id()`